### ISSUE TITLE
Can't provision a new computer from a phone without a working camera

### ISSUE THREAD

**Issue by Brianetta** [keybase/keybase-issues#3606]
When provisioning a computer from an Android mobile phone:

- My computer provided a barcode, and some key words
- My phone provided a barcode, activated the camera, and offered some key words
- Neither offered any means by which to type in key words

Without a working camera, there was no way to provision the computer from the mobile phone.
Clients were 4.5 (on computer) and 4.6 (on phone).

**Comment by buoyad** [comment]
Can you reproduce this on the latest versions of the app, making sure to choose the right device types on both ends?

**Comment by Brianetta** [comment]
Yes, I can. Version is 4.7.1 on both devices. The process remains exactly as described.

**Comment by bjorn** [comment]
I just ran into a similar issue, using the latest 5.0.0 version. In my case, both my phone and my desktop client wanted me to type in the words. I think in my case the bug is solely with the desktop client:

1. Go to Devices
2. Click "Add device or paper key"
3. Add a phone
4. Notice a QR code is shown (the encoded 9 words secret)
5. Click "Type secret instead"
6. Notice that now the desktop client doesn't show the 9 words secret, but _asks for it instead!_

With step 5 causing the desktop client to ask for the words instead of showing them in plain text, I'm unable to provide my phone client with the words without using the camera.

**Comment by Brianetta** [comment]
@bjorn was provisioning the phone from the computer, I was provisioning the computer from the phone. It would seem that both cases are a little over-dependent on a working camera.
